backend/src/main/kotlin/rocks/didit/sefilm/Oauth2.kt

```diff
@@ -152,7 +152,8 @@ class OpenIdConnectFilter(defaultFilterProcessesUrl: String,
                         lastName = principal.lastName,
                         nick = principal.lastName ?: "Houdini",
                         email = principal.username ?: "",
-                        avatar = principal.avatarUrl)
+                        avatar = principal.avatarUrl,
+                        foretagsbiljetter = emptyList())
```

Comment:
This does most definitely not belong here. This is only for information that Google or some other auth provider provides. Consider moving it to either the user object or some other place, but it shouldn't be stored in the SecurityContext/Session